JavaScript/test/runner-datetime.js

```diff
@@ -210,9 +210,14 @@ function getModel(config) {
     return findModel(options, cultureCode);
 }
 
+function parseISOLocal(s) {
```

Comment:
Can we add a comment in the code here to explain why this was needed? It will be helpful in the future.